packages/inspector/src/add/add-ai-agent.ts

```
@@ -0,0 +1,194 @@
+                `  Then use: ${propName}: ${schemaName}`
+            )
+          }
+        }
+      }
+    }
+
+    const middleware = resolveMiddleware(state, obj, tags, checker)
+    const permissions = resolvePermissions(state, obj, tags, checker)
+
+    state.serviceAggregation.usedFunctions.add(nameValue)
+    extractWireNames(middleware).forEach((name) =>
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)